README.md

```diff
@@ -47,30 +49,106 @@ Requirements
 Installation
 ------------
 
-* Checkout or link the code in this repo to
-`/usr/lib/archivematica/automation-tools`
-* Create virtualenv `/usr/share/python/automation-tools` and pip install
-requirements there
-* Create directories `/var/log/archivematica/automation-tools` and
-`/var/archivematica/automation-tools` owned by user `archivematica`, for
-log/database/PID files.
-* Create directory `/etc/archivematica/automation-tools` and add configuration
-files there. Files in the `etc/` directory of this repository can be used as an
-example (also see below for more about configuration)
+Follow each of the steps below to install automation-tools:
+
+1. Make the following directories
+
+```bash
+mkdir -p /usr/lib/archivematica/automation-tools &&
+mkdir /usr/share/python/automation-tools &&
+mkdir -p /var/log/archivematica/automation-tools &&
+mkdir /var/archivematica/automation-tools &&
+mkdir /etc/archivematica/automation-tools
+```
+
+1. Change ownership of the directories to be owned by archivematica (group
+archivematica)
+
+```bash
+chown archivematica:archivematica /var/log/archivematica/automation-tools &&
+chown archivematica:archivematica /var/archivematica/automation-tools
+```
+
+1. Clone this automation-tools repository into your `usr/lib`
+
+```bash
+cd /usr/lib/archivematica/automation-tools/
+git clone https://github.com/artefactual/automation-tools.git .
+```
+
+1. Set up Python's virtual environment
+
+```bash
+cd /usr/share/python/automation-tools
+virtualenv venv
+source venv/bin/activate
+pip install -r /usr/lib/archivematica/automation-tools/requirements.txt
+```
+
+1. (Optional) Update and retrieve required packages
+
+```bash
+apt-get update
+apt-get install p7zip-full
+```
+
+You may also want/need to `apt-get install sudo && apt-get install vim`.
+
+To use automation-tools, `cp` the scripts you want to use into your
+`/etc/archivematica/automation-tools` directory.
```

Comment:
Perhaps we can break up the text here with a further sub-heading:

`### Configuring the automation tools to run`

I'm not sure if that would be a H2 (##) or H3 (###) or what the resturcturedText version of that looks like. 

We can then rework this sentence to remove the abbreviated `cp` and replace it with copy, so that we follow the pattern you've established elsewhere to say: 'this is what you're going to do' followed by 'this is the command you're going to use to do that'. Which I feel is good practice to capture a broad reader-type. 

Perhaps line `100: Follow the steps below as an example for setting up a script.` can form part of an introductory paragraph to what the reader is doing next.